modules/virt-ssh-configuration-considerations.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-ssh.adoc
+
+:_content-type: REFERENCE
+[id="virt-ssh-configuration-considerations_{context}"]
+= SSH configuration considerations
```

Comment:
@phoracek Can you please review this file?

Reply:
I changed the file name. I will add review request to updated file.